llvm/utils/lit/tests/Inputs/shtest-cat/cat-v.txt

```diff
@@ -0,0 +1,49 @@
+# C Program to generate all ASCII characters in allchars.txt
+# #include <stdio.h>
+# int main() {
+#     FILE *fptr = fopen("allchars.txt", "w");
+#     if(fptr == NULL) {
+#         return 1;
+#     }
+#     for(int i = 0; i < 256; i++) {
+#         fprintf(fptr, "%c", i);
+#     }
+#     fclose(fptr);
+#     return 0;
+# }
+
+
+## Tests to check cat -e output for ASCII characters and newlines
+
+# RUN: cat -v %S/txt-files/allchars | FileCheck %s --check-prefix=ALLCHARS
+# RUN: cat -v %S/txt-files/newline | FileCheck %s --check-prefix=NEWLINE
+
```

Comment:
Comments here, similar to the one for `-e` would help

Reply:
done